src/tags/index.js

```diff
@@ -95,19 +95,17 @@ module.exports = function(fractal){
 
                     if (token.withStack !== undefined) {
                         passedArguments = Twig.expression.parse.apply(this, [token.withStack, context]);
-
-                        _.forEach(passedArguments, function (value, name) {
-                            // It makes no sense to pass variables that are not
-                            // supported by the component.
-                            if (innerContext[name] === undefined) {
-                                return;
-                            }
-                            if (name.indexOf('attributes') > -1) {
-                                innerContext[name].merge(value);
-                            }
-                            else {
-                                innerContext[name] = value;
-                            }
+                        _.forEach(innerContext, function (value, name) {
```

Comment:
The previously existing comment could be retained for clarity:
```suggestion
                        // Variables not defined by the component context are
                        // intentionally ignored.
                        _.forEach(innerContext, function (value, name) {
```